doc/content/demonstration/index.md

```diff
@@ -0,0 +1,16 @@
+# MALAMUTE Demonstration Cases
+
+Documentation regarding demonstration problems using MALAMUTE are outlined here.
+
+## Verification & Validation Problems
+
+!style halign=left
+Problems and documentation that describe the basis of [MALAMUTE verification & validation](malamute_vvr.md)
+are described in this section.
+
+1. [verification/thermal_contact_verification.md]
```

Comment:
Would a title before the link be appropriate here? I can't quickly find the rendered documentation page to see what this code produces

Reply:
Please disregard. I was able to find the rendered page; this change looks fine to me as is